INTITA/Modules/OpportunitiesScreen/OpportunitiesViewModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  OpportunitiesViewModel.swift
+//  INTITA
+//
+//  Created by Viacheslav Markov on 24.01.2021.
+//
+
+import Foundation
+
+class OpportunitiesViewModel {
```

Comment:
According to MVVM, it should have a link to OpportunitiesModel.

Reply:
Thanks for advice. During the improvement of functionality the OpportunitiesViewModel was deleted.